src/extension/features/budget/budget-category-features/display-target-goal-amount/settings.js

```diff
@@ -9,7 +9,7 @@ module.exports = {
   options: [
     { name: 'Do not display goal amount (default)', value: '0' },
     { name: 'Display goal amount and warn of overbudget with red', value: '1' },
-    { name: 'Display goal amount but show overbudget as green', value: '2' },
+    { name: 'Display goal amount and show funded goals as green', value: '2' },
```

Comment:
I think these should be:

```
    { name: 'Display goal amount and highlight overfunding as red', value: '1' },
    { name: 'Display goal amount and highlight overfunding as green', value: '2' },
```

Reply:
If you get to this before me I'll merge, otherwise I'll merge when I can get around to making the change alongside your PR. Thanks!

Reply:
There's a slight difference in behaviour between the two settings that I was trying to highlight, where the red highlights on greater than, while the green highlights on greater than _or_ equals.
I'm happy to go with those descriptions as I think it's fairly intuitive, but I wanted to make it clearer as the user in the original issue was a little confused by that.

Reply:
I see, this makes sense I think. I wonder if a third option should be considered but for now this works.